readme.md

```diff
@@ -85,23 +91,28 @@ msbuild openrct2.proj /t:g2
 msbuild openrct2.proj /t:PublishPortable
 ```
 
-### Mac:
-We support native builds for macOS (limited to i386 only for now).
-Make sure that you have [Homebrew](http://brew.sh/) installed and than run the following commands to install all the needed libraries and build OpenRCT2.
-```
-# Install libraries
-./install.sh
+### macOS:
+#### Xcode:
+The recommended way of building OpenRCT2 for macOS is with Xcode. The Xcode build will create a self-contained application bundles which include all the necessary game files and dependencies. Open the project file OpenRCT2.xcodeproj in Xcode and build from there. Building this way will handle the dependencies for you automatically. You can also invoke an Xcode build from the command line using `xcodebuild`.
 
-# Build OpenRCT2
-./build.sh
+#### CMake:
+A command line version of OpenRCT2 can be built using CMake. This type of build requires you to provide the dependencies yourself. The supported method of doing this is with [Homebrew](http://brew.sh). Once you have Homebrew installed, you can download all the required libraries with this command:
```

Comment:
I still don't think it is valid description. From a perspective of a user reading this readme, I would assume this is a different kind of build than Xcode one. As in: Xcode is not able of performing any command line interactions, which simply is not true.

It would be better if you simply pointed out that this does not build "self-contained application bundles", but do check it is the case with `make package`.